recipes/mosquitto/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "mosquitto" %}
+{% set version = "2.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://mosquitto.org/files/source/{{ name }}-{{ version }}.tar.gz
+  sha256: d0dde8fdb12caf6e2426b4f28081919a2fce3448773bdb8af0d3cd5fe5776925
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+
+  host:
+    - openssl
+    - libxslt
+
+
+test:
+  commands:
+    - mosquitto_sub --help || true  # [not win]
+
+about:
+  home: https://mosquitto.org/
+  license: EPL-2.0
+  license_file: epl-v20
+  summary: 'Eclipse Mosquitto is an open source (EPL/EDL licensed) message broker\
+ that implements the MQTT protocol versions 5.0, 3.1.1 and 3.1. '
```

This should fit on one line, how about:
```suggestion
  summary: 'An open source message broker that implements the MQTT protocol.'
```